public/config.json

```diff
@@ -34,7 +36,10 @@
     "api_key": "abcdefg",
     "index_name": "core_data_places",
     "query_by": "name",
-    "default_sort": "name"
+    "default_sort": "name",
+    "overrides": {
+      "geoLocationField": "coordinates"
+    }
```

Comment:
Should this be configurable in tina?

Reply:
I'm not sure it's something we'd want in Tina, as this will likely be set by the developer setting up the site, not changed by the project owner. There should be [documentation](https://github.com/performant-software/core-data-places/blob/314ccdeb3d3520f848c53979f0ba6fe8b81ad56a/README.md?plain=1#L63) that links the the Typesense adapter we're using in `README.md`.